providers/standard/tests/unit/standard/utils/test_python_virtualenv.py

```diff
@@ -24,7 +24,12 @@
 import pytest
 
 from airflow.providers.standard.utils.python_virtualenv import _generate_pip_conf, _use_uv, prepare_virtualenv
-from airflow.utils.decorators import remove_task_decorator
+
+try:
+    from airflow.sdk.bases.decorator import remove_task_decorator
+except ImportError:
+    # Fallback for Airflow < 3.1
+    from airflow.utils.decorators import remove_task_decorator  # type: ignore[no-redef]
```

Comment:
Could you add this to the version_compat module and import it?

Reply:
Since this is a test, the version_compat under `devel-common/src/tests_common/test_utils/version_compat.py`